player/tests.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from .models import Player, BallGame
+import pytest
+
+
+@pytest.mark.django_db
+class TestPlayerModel:
+
+    @pytest.fixture
+    def user(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def saved_player(self, user):
```

Comment:
calling Player.objects.create() doesn't save the instance to the db ?

Reply:
I suggest adding a test for fetching the created instance from the db and making sure it's the same as the one in the fixture. If it's saved automatically, it will pass. Otherwise, save it to make it pass.

Reply:
done